README.md

```diff
@@ -1,2 +1,131 @@
+[ci-img]: https://img.shields.io/travis/ciena-frost/ember-frost-demo-components.svg "Travis CI Build Status"
+[ci-url]: https://travis-ci.org/ciena-frost/ember-frost-demo-components
+
+[cov-img]: https://img.shields.io/coveralls/ciena-frost/ember-frost-demo-components.svg "Coveralls Code Coverage"
+[cov-url]: https://coveralls.io/github/ciena-frost/ember-frost-demo-components
+
+[npm-img]: https://img.shields.io/npm/v/ember-frost-demo-components.svg "NPM Version"
+[npm-url]: https://www.npmjs.com/package/ember-frost-demo-components
+
+[![Travis][ci-img]][ci-url] [![Coveralls][cov-img]][cov-url] [![NPM][npm-img]][npm-url]
+
 # ember-frost-demo-components
+
 A collection of components and blueprints for showcasing frost components
+
+ * [Installation](#installation)
+ * [Getting Started](#getting-started)
+ * [Components](#components)
+ * [Blueprints](#blueprints)
+ * [Contributing](#development)
+
+
+## Installation
+```
+ember install ember-frost-demo-components
+```
+
+## Getting Started
+
+1. This *addon* installs `CodeMirror` and `Prism` for source editing and syntax highlighting.  You must update `ember-cli-build.js` to include options for those *addons* to bring in necessary themes.
+
+  ```js
+  var app = EmberApp(defaults, {
+    codemirror: {
+      modes: ['javascript'],
+      // frost-demo-editor defaults to 'mdn-like'
+      themes: ['mdn-like']
+    },
+    'ember-prism': {
+      // whatever your preference
+      theme: 'coy',
+      components: ['javascript']
+    }
+    ...
+  })
+  ```
```

Comment:
I believe you can remove the need for this manual step by doing something similar to https://github.com/rwjblue/ember-computed-decorators#setup-with-addon